api/src/main/java/com/valtech/aem/saas/api/query/SimpleFilter.java

```diff
@@ -15,15 +16,24 @@
 public class SimpleFilter implements Filter {
 
     private static final String FILTER_FIELD_VALUE_DELIMITER = ":";
+    public static final String QUOTE = "\"";
+    public static final String REGEX_MATCH_NON_WHITESPACE = "\\S+";
 
     private String name;
     private String value;
 
     @Override
     public String getQueryString() {
         if (StringUtils.isNoneBlank(name, value)) {
-            return String.join(FILTER_FIELD_VALUE_DELIMITER, name, value);
+            return String.join(FILTER_FIELD_VALUE_DELIMITER, name, getSafeValue(value));
         }
         return StringUtils.EMPTY;
     }
+
+    private String getSafeValue(@NonNull String value) {
+        if (!value.matches(REGEX_MATCH_NON_WHITESPACE)) {
```

Comment:
instead please use simply StringUtils.containsWhitespace() to catch all cases

Reply:
ah, i was looking alike method in StringUtils, but I did not notice that one. thx